src/fragments/lib/geo/android/getting_started/20_cli_resources.mdx

```diff
@@ -1,22 +1,6 @@
 > Prerequisite: [Install and configure the Amplify CLI](/cli/start/install)
 
-The primary way to provision Geo resources is through the Amplify CLI. You can use the following command to install this globally.
-
-```sh
-npm i -g @aws-amplify/cli@geo
-```
-
-Now, let's make sure that the right version was installed:
-
-```sh
-amplify version
-```
-
-That should show a version with the `-geo.x` tag, for example, `6.0.1-geo.0`.
-
-> **Note:** Make sure that version `6.0.1-geo.0` or above is installed.
-
-Once that is complete, you can run the following command from your project's root folder to initialize Amplify in your repo:
+To start provisioning Geo resources in the backend, go to your project directory and execute the command.
```

Comment:
nit: replace "." with ":" at the end of the sentence

Reply:
good callout, thanks. addressed in changes.